docs/userguide.md

```diff
@@ -12,10 +12,9 @@ is a detailed list of binaries required.
 
 The federation deployment depends on `kubebuilder`, `etcd`, `kubectl`, and
 `kube-apiserver` >= v1.11 being installed in the path. The `kubebuilder`
-release packages all of these dependencies together, but the latest version of
-`kubebuilder` (v1.0.0 as of this writing) only includes binaries for
-kubernetes 1.10. Once `kubebuilder` bundles binaries >= 1.11, it will
-only require downloading the `kubebuilder` release.
+([v1.0.3](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1.0.3)
+as of this writing) release packages all of these dependencies together and
```

Comment:
Please end the sentence at 'together' - I think the remainder is redundant.

Reply:
Done